lib/rule-tester/rule-tester.js

```diff
@@ -304,6 +304,17 @@ function throwForbiddenMethodError(methodName, prototype) {
     };
 }
 
+const messagePlaceholderRegex = /\{\{([^{}]+?)\}\}/gu;
+
+/**
+ * Extracts names of missing {{ placeholders }} from the reported message.
+ * @param   {string} message Reported message
+ * @returns {string[]} Array of missing placeholder names
+ */
+function getMissingMessagePlaceholders(message) {
```

Comment:
Does the concept of "missing" actually come into play in this function? Or is just looking for placeholders in the message? If it's just looking for placeholders, which happen to be the missing ones in the context where it's called, then the function could be more generic by removing "missing" from its name.